platform/core.multitabs/src/org/netbeans/core/multitabs/impl/ButtonPopupSwitcher.java

```diff
@@ -355,6 +354,10 @@ private boolean changeSelection( KeyEvent event ) {
                         selRow = Math.min( pTable.getModel().getRowCount()-1, selRow );
                         selCol = Math.min( pTable.getModel().getColumnCount()-1, selCol );
                         switched = true;
+                    }else if( tabIndex == -1 ){
```

Comment:
Should have space before "else" and after "if". Some similar formatting errors elsewhere. (Don't worry about fixing formatting issues in existing code, though.)

Reply:
Why does tabIndex == -1 indicate that closeSelectedDocumentList should be invoked? Would probably be cleaner to check if item.isSeparator(). Nice that you're handling this keypress, though!

Reply:
You are right, thought I would keep the conditions similiar, but reads much better when it checks if item.isSeparator().